lua/entities/gmod_tardis/modules/sv_power.lua

```diff
@@ -1,5 +1,21 @@
--- Power
+-- Power Exterior
 
-function ENT:TogglePower()
-    self.interior:TogglePower()
+ENT:AddHook("Initialize","power-init", function(self)
+    self:SetData("power-state",true,true)
+end)
+
+if SERVER then
+    function ENT:TogglePower()
+        local on = not self:GetData("power-state",false)
+        self:SetPower(on)
+    end
+    function ENT:SetPower(on)
+        if not (self:CallHook("CanTogglePower")==true or self.exterior:CallHook("CanTogglePower")==true) then return end
```

Comment:
Inverse hook so that it only stops if CanTogglePower hook returns false. nil and true should count as true

Like this but you probably need to change some other code too
`if self:CallHook("CanTogglePower")==false or self.exterior:CallHook("CanTogglePower")==false then return end`